scripts/run.sh

```diff
@@ -5,20 +5,22 @@
 # function to clean up shared memory
 function cleanup() {
     sleep 0.5
-    cd shm_wrapper && ./shm_stop
+    cd bin && ./shm_stop
     exit 0
 }
 
-cd shm_wrapper && ./shm_start &
+cd bin && ./shm_start &
 trap cleanup INT # sets cleanup as Ctrl-C handler
 sleep 0.5
 
 # starts all three Runtime processes in the background
-cd net_handler && ./net_handler &
+cd bin && ./net_handler &
 sleep 0.5
-cd executor && ./executor &
+cd bin && ./executor &
 sleep 0.5
-cd dev_handler && ./dev_handler &
+cd bin && ./dev_handler &
+sleep 0.5
+cd bin && ./network_switch & 
 
```

Comment:
These changes guaranteed we clean the executables not in the `runtime/bin` directories from devices with older installations of Runtime.